config/presets/identity/email-password-interaction-policy.json

```diff
@@ -0,0 +1,216 @@
+      "EmailPasswordResourceStoreStorageAdapter:_args_storagePathname": "/idp/email-password-db",
+      "EmailPasswordResourceStoreStorageAdapter:_args_saltRounds": 10,
+      "EmailPasswordResourceStoreStorageAdapter:_args_store": {
+        "@id": "urn:solid-server:default:ResourceStore"
+      }
+    },
+    {
+      "@id": "urn:solid-server:email-password-interaction:OidcInteractionCompleter",
+      "@type": "OidcInteractionCompleter"
+    },
+    {
+      "@id": "urn:solid-server:email-password-interaction:LoginRenderHandler",
+      "@type": "RenderEjsHandler",
+      "RenderEjsHandler:_viewsFolder": {
+        "@id": "urn:solid-server:default:variable:viewsFolder"
+      },
+      "RenderEjsHandler:_ejsTemplatePath": "./email-password-interaction/login.ejs"
+    },
+    {
+      "@id": "urn:solid-server:email-password-interaction:RegisterRenderHandler",
+      "@type": "RenderEjsHandler",
+      "RenderEjsHandler:_viewsFolder": {
+        "@id": "urn:solid-server:default:variable:viewsFolder"
+      },
+      "RenderEjsHandler:_ejsTemplatePath": "./email-password-interaction/register.ejs"
+    },
+    {
+      "@id": "urn:solid-server:email-password-interaction:ForgotPasswordRenderHandler",
+      "@type": "RenderEjsHandler",
+      "RenderEjsHandler:_viewsFolder": {
+        "@id": "urn:solid-server:default:variable:viewsFolder"
+      },
+      "RenderEjsHandler:_ejsTemplatePath": "./email-password-interaction/forgotPassword.ejs"
+    },
+    {
+      "@id": "urn:solid-server:email-password-interaction:ResetPasswordRenderHandler",
+      "@type": "RenderEjsHandler",
+      "RenderEjsHandler:_viewsFolder": {
+        "@id": "urn:solid-server:default:variable:viewsFolder"
+      },
+      "RenderEjsHandler:_ejsTemplatePath": "./email-password-interaction/resetPassword.ejs"
+    },
+    {
+      "@id": "urn:solid-server:email-password-interaction:EmailSentRenderHandler",
+      "@type": "RenderEjsHandler",
+      "RenderEjsHandler:_viewsFolder": {
+        "@id": "urn:solid-server:default:variable:viewsFolder"
+      },
+      "RenderEjsHandler:_ejsTemplatePath": "./email-password-interaction/emailSent.ejs"
+    },
+    {
+      "@id": "urn:solid-server:email-password-interaction:MessageRenderHandler",
+      "@type": "RenderEjsHandler",
+      "RenderEjsHandler:_viewsFolder": {
+        "@id": "urn:solid-server:default:variable:viewsFolder"
+      },
+      "RenderEjsHandler:_ejsTemplatePath": "./email-password-interaction/message.ejs"
+    },
+    {
+      "@id": "urn:solid-server:email-password-interaction:InteractionHttpHandler",
+      "@type": "ErrorHandlingWaterfallHandler",
+      "ErrorHandlingWaterfallHandler:_onErrorHandler": {
+        "@type": "BasicOnErrorHttpHandler"
+      },
+      "ErrorHandlingWaterfallHandler:_handlers": [
+        {
+          "@type": "RouterHandler",
+          "RouterHandler:_allowedMethods": [ "GET" ],
+          "RouterHandler:_allowedPathnames": [ "^\/interaction\/[_A-Za-z0-9\\-]+\/?$" ],
```

Comment:
That's a weird path name, we definitely need something more specific, such as `/idp/`.

Reply:
Changed to `/idp/interaction/:uid/`